Sunrise/ViewControllers/ProductOverviewViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import UIKit
+import ReactiveCocoa
+import Result
+import SDWebImage
+
+class ProductOverviewViewController: UICollectionViewController {
+
+    var viewModel: ProductOverviewViewModel? {
+        didSet {
+            dispatch_async(dispatch_get_main_queue()) { [unowned self] in
+                self.bindViewModel()
+            }
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // After implementing POP search, view model will be instantiated with initial query
+        viewModel = ProductOverviewViewModel(limit: 15)
+    }
+
+    // MARK: - Bindings
+
+    private func bindViewModel() {
+        guard let viewModel = viewModel else { return }
+
+        navigationItem.title = viewModel.title
+
+        viewModel.isLoading.producer
+        .observeOn(UIScheduler())
+        .startWithNext({ [weak self] isLoading in
+            if !isLoading {
+                self?.collectionView?.reloadData()
+            }
+        })
+
+        viewModel.alertMessageSignal
+        .observeOn(UIScheduler())
+        .observeNext({ [weak self] alertMessage in
+            let alertController = UIAlertController(
+            title: "Oops!",
+                    message: alertMessage,
+                    preferredStyle: .Alert
+            )
+            alertController.addAction(UIAlertAction(title: "OK", style: .Cancel, handler: nil))
+            self?.presentViewController(alertController, animated: true, completion: nil)
+        })
+    }
+
+
+    // MARK: - Navigation
+
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        if let selectedCell = sender as? ProductOverviewCell, indexPath = collectionView?.indexPathForCell(selectedCell),
+                productViewController = segue.destinationViewController as? ProductViewController, viewModel = viewModel {
+            let productDetailsViewModel = viewModel.productDetailsViewModelForProductAtIndexPath(indexPath)
```

Comment:
Since we also have `productPriceAtIndexPath` or `productNameAtIndexPath` in this file, couldn't we simply rename `productDetailsViewModelForProductAtIndexPath` to `productViewModelAtIndexPath` for consistency and readability?